docs/Account sign in (Account Protection) - Approved and rejected sign in flows.md

```diff
@@ -0,0 +1,88 @@
+# Microsoft Dynamics 365 Fraud Protection - API examples
+## Account sign in (Account Protection APIs) - Approved and rejected sign in flows
+
+After sending Dynamics 365 Fraud Protection a sign in event, use your merchant rule decision returned by Dynamics 365 Fraud Protection to either continue with or stop your sign in workflow.
+
+## Helpful links
+- [Calling Dynamics 365 Fraud Protection](./Authenticate&#32;and&#32;call&#32;Fraud&#32;Protection.md)
+- [SignIn - Data model and endpoint](https://apidocs.microsoft.com/services/dynamics365fraudprotection#/v1.0/V1.0MerchantservicesEventsSignUpPost)
```

Comment:
Waiting on Swagger updates to update this link

Reply:
Removed for now.  Will re-add later.